tests/Functional/Schema/SynapseSchemaReflectionTest.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Keboola\TableBackendUtils\Functional\Schema;
+
+use Keboola\TableBackendUtils\Column\ColumnCollection;
+use Keboola\TableBackendUtils\Column\SynapseColumn;
+use Keboola\TableBackendUtils\Schema\SynapseSchemaReflection;
+use Keboola\TableBackendUtils\Table\SynapseTableQueryBuilder;
+use Tests\Keboola\TableBackendUtils\Functional\SynapseBaseCase;
+
+/**
+ * @covers SynapseSchemaReflection
+ */
+class SynapseSchemaReflectionTest extends SynapseBaseCase
```

Comment:
Já bych byl tady v těch testech striktnější.
Měl bys mít i jiné testovací schema s tabulkama a views, abys ověřil že ti to skutečně vrací pouze z toho tvého.
Kdyby DB byla prázdná a tys zapomněl na podmínku v query, tak ti to taky vrátí všechny z celý databáze.

Reply:
Přidal sem nějaký hardening, že to fakčí jak má, ale na tu podmínku v query zapomenou nejde, protože je fixní v té Reflection class

Reply:
super